README.md

```diff
@@ -98,6 +98,14 @@ The site is configured to use
 startup time while Rails loads. See the Zeus github page for
 usage instructions.
 
+Once [npm dependencies are
+installed](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Karma), AngularJS tests can be run with:
+
+    rake karma:run
```

Comment:
Shouldn't we call it in the right environment?

    rake karma:run RAILS_ENV=test

That would load twice as fast.